src/analysis/retail/priest/shadow/modules/spells/Voidform.tsx

```diff
@@ -60,10 +96,10 @@ class Voidform extends Analyzer {
       </>
     );
     let value = QualitativePerformance.Good;
-    if (extension <= 15) {
+    if (extension <= 15.1) {
       value = QualitativePerformance.Ok;
     }
-    if (extension <= 10) {
+    if (extension <= 10.1) {
```

Comment:
whats the story here?

Reply:
I had just been testing some other values while looking at logs, and didn't change them back properly.